packages/help-center/src/components/help-center-header.tsx

```diff
@@ -5,20 +5,29 @@ import { useI18n } from '@wordpress/react-i18n';
 import clsx from 'clsx';
 import { useCallback } from 'react';
 import { Route, Routes, useLocation } from 'react-router-dom';
+import { SUPPORT_BLOG_ID } from '../constants';
+import { usePostByKey } from '../hooks/use-post-by-key';
+import { useSupportArticleAlternatePostKey } from '../hooks/use-support-article-alternates-query';
 import { HELP_CENTER_STORE } from '../stores';
 import type { Header } from '../types';
 import type { HelpCenterSelect } from '@automattic/data-stores';
 
 export function ArticleTitle() {
-	const location = useLocation();
 	const { search } = useLocation();
 	const params = new URLSearchParams( search );
-	const title = location.state?.title || params.get( 'title' ) || '';
+	const postId = params.get( 'postId' );
+	const blogId = params.get( 'blogId' );
+
+	const postKey = useSupportArticleAlternatePostKey(
+		+( blogId ?? SUPPORT_BLOG_ID ),
+		+( postId || 0 )
```

Comment:
I think `useSupportArticleAlternatePostKey` should have default `blogId` itself.